src/main/java/me/mrCookieSlime/Slimefun/api/SlimefunItemStack.java

```diff
@@ -245,4 +246,20 @@ else if (PatternUtils.ALPHANUMERIC.matcher(texture).matches()) {
         }
     }
 
+    /**
+     * Adds additional lores for the {@link ItemStack}.
+     *
+     * @param newLore
+     *                New lores to be added to the {@link ItemStack}
+     */
+    public void addLore(String... newLore) {
```

Comment:
I don't really think this method is useful here, I'd argue this should be removed and just done manually or via the Consumer in the constructor.